docs/changelog.md

```diff
@@ -15,6 +15,7 @@ dev
 - [bugfix] Hide cursor while pipx runs
 - [feature] Add environment variable `USE_EMOJI` to allow enabling/disabling emojies (#376)
 - [refactor] Moved all commands to separate files within the commands module (#255).
+- [feature] `list` now has a new option `--include-injected` to show the injected packages in the main apps
```

Comment:
This should be moved to the `dev` section since it was not included in a release. Otherwise, this looks good.

Reply:
I have moved it to the correct position.